### PR TITLE
Component prop types

### PR DESCRIPTION
**Pull request by kulakowka**
Added prop types to: 
- Admin
- AdminRoutes
- CrudRoute
- Resource

### CODE REVIEW

File: src/Admin.js [hunk 1]
```diff
@@ -90,7 +90,7 @@ const Admin = ({
     );
 };
 
-const componentPropType = PropTypes.oneOfType([PropTypes.func, PropTypes.string]);
+const componentPropType = PropTypes.node;
```
Comment (fzaninotto) [inline src/Admin.js]: Not good: `PropTypes.node` is for *elements*, not *components*

File: src/AdminRoutes.js [hunk 2]
```diff
@@ -46,4 +46,10 @@ const AdminRoutes = ({ customRoutes, resources = [], dashboard }) => (
     </Switch>
 );
 
+AdminRoutes.propTypes = {
+    customRoutes: PropTypes.array,
+    resources: PropTypes.array,
+    dashboard: PropTypes.node,
```
Comment (fzaninotto) [inline src/AdminRoutes.js]: nope

File: src/CrudRoute.js [hunk 3]
```diff
@@ -38,4 +38,16 @@ const CrudRoute = ({ resource, list, create, edit, show, remove, options }) => {
     );
 };
 
+const componentPropType = PropTypes.node;
```
Comment (fzaninotto) [inline src/CrudRoute.js]: nope

File: src/Resource.js [hunk 4]
```diff
@@ -1,7 +1,7 @@
 import React, { PropTypes } from 'react';
 import ViewListIcon from 'material-ui/svg-icons/action/view-list';
 
-const componentPropType = PropTypes.oneOfType([PropTypes.func, PropTypes.string]);
+const componentPropType = PropTypes.node;
```
Comment (fzaninotto) [inline src/Resource.js]: nope

File: src/Admin.js (hunk 1 above)
Comment (kulakowka) [inline src/Admin.js]: Thanks for the explanation. I'll fix it.

**Comment by fzaninotto** [comment]
Bump: please update this PR soon, or we'll close it.

**Comment by fzaninotto** [comment]
No news for a month, closing.

**Pull request closed without merging**